RxCocoa/iOS/UIPickerView+Rx.swift

```diff
@@ -40,6 +61,20 @@
                 }
             return ControlEvent(events: source)
         }
+        
+        public func items<O: ObservableType,
```

Comment:
It would be great to add a simple example in RxExample app. Perhaps something similar to `SimpleTableViewExampleSectionedViewController`.